internal/testrunner/runners/pipeline/runner.go

```diff
@@ -33,9 +34,15 @@ const (
 	TestType testrunner.TestType = "pipeline"
 )
 
+var (
+	serverlessEnableCompareResults = environment.WithElasticPackagePrefix("SERVERLESS_PIPELINE_TEST_ENABLE_COMPARE_RESULTS")
```

Comment:
I am thinking now that maybe we should leave this enabled by default, so when running locally, by default we see the errors. And we can disable in the CI jobs where we want to use this. WDYT?
```suggestion
	serverlessEnableCompareResults = environment.WithElasticPackagePrefix("SERVERLESS_PIPELINE_TEST_DISABLE_COMPARE_RESULTS")
```